doc/classes/Theme.xml

```diff
@@ -405,6 +432,17 @@
 				Returns [code]false[/code] if the theme does not have [code]theme_type[/code].
 			</description>
 		</method>
+		<method name="is_type_variation" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="theme_type" type="StringName">
+			</argument>
+			<argument index="1" name="base_type" type="StringName">
+			</argument>
+			<description>
+				Returns [code]true[/code] if in this [Theme] [code]theme_type[/code] is marked as a variation of [code]base_type[/code].
```

Comment:
"Theme theme_type" sounds odd. Shouldn't it be "[Theme]'s theme_type"? Or maybe [Theme ] is not necessary here?

Reply:
Yeah, I can probably drop "in the Theme", if it reduces readability.